medusa/restore_cluster.py

```diff
@@ -235,8 +236,8 @@ def _chunk(my_list, nb_chunks):
             target_tokens = {_tokens_from_ringitem(ringitem): host for host, ringitem in target_tokenmap.items()}
             backup_tokens = {_tokens_from_ringitem(ringitem): host for host, ringitem in tokenmap.items()}
 
-            target_tokens_per_host = set(_token_counts_per_host(tokenmap))
-            backup_tokens_per_host = set(_token_counts_per_host(target_tokenmap))
+            target_tokens_per_host = _token_counts_per_host(tokenmap)
+            backup_tokens_per_host = _token_counts_per_host(target_tokenmap)
```

Comment:
Ok, so what was done before was actually broken. Considering all nodes have the same number of tokens good enough for now.